test/controllers/foreman/organizations_controller_test.rb

```diff
@@ -29,4 +29,15 @@ def test_use_newly_created_org
     assert_redirected_to step2_organization_path(:id => org.to_param)
     assert_equal session[:organization_id], org.id
   end
+
+  test "should not be able to auto attach when host is in simple content access mode" do
+    organization = FactoryBot.create(:organization)
+    organization.stubs(:simple_content_access?).returns(true)
+
+    if organization.simple_content_access?
```

Comment:
We don't need this `if` since you are stubbing simple content access to true. We want to make this assertion always.

This test doesn't appear to be issuing any http request so it's not actually testing anything. You'll want to add your test to this file instead: `test/controllers/api/v2/organizations_controller_test.rb`. You'll see the other tests for auto attach there